skt/__init__.py

```diff
@@ -428,7 +430,10 @@ def bisect_iter(self, bad):
                 if m:
                     binfo = m.group(1)
                     ret = 1
-                    logging.warning("Bisected, bad commit: %s" % binfo)
+                    logging.warning(
+                        "Bisected, bad commit: %s",
+                        binfo
+                    )
```

Comment:
Can we not expand the lines this way if it's not needed? When the lines are this short, it doesn't help readability :/